Apps/VisionOS/VisionOS/AppDelegate.swift

```diff
@@ -0,0 +1,39 @@
+import CioTracking
+import UIKit
+
+class AppDelegate: NSObject, UIApplicationDelegate {
+    func application(
+        _ application: UIApplication,
+        didFinishLaunchingWithOptions launchOptions: [UIApplication
+            .LaunchOptionsKey: Any]? = nil
+    ) -> Bool {
+        /*
+
+         How do I get the values for the siteId and apiKy?
+         1- Login or Signup in CustomerIO platform: https://fly.customer.io/
+         2- On the top right section click the
+         Settings Icon >> Workspace Settings >> API and webhook credentials
+         You will see list of one or more workspaces and each has
+         its own pair of siteId and apiKey, these are the ones
+         to use here.
+         */
+
+        let workspaceSettings = AppState.shared.workspaceSettings
+        if workspaceSettings.isSet() {
```

Comment:
What happens if workspace settings are *not* set? 

At a minimum, should we create a log warning or something?

Reply:
Not really. The goal here is not to ensure full setup ready for every app run; the gaol is to ensure the user don't have to go to the initialize screen and call initialize manually. More of a UX piece instead of production-ready setup. At the end of the day, it is a sample app meant to make it easy to see the SDK usage in action instead of being in-itself a production ready app

Reply:
But one thing I found missing while working on this part, is that the callsite have noway to tell if they enter wrong credentials. The SDK won't report anything, as far as I know at least. Something we might want to revise at some point later.